### DISCUSSION TITLE
Confusion Between Vue.js Components and Views: Seeking Guidance for Thesis Project

### DISCUSSION THREAD

**Discussion by chanezAMK** [vuejs/router#2180] [Help and Questions]
Hello,I'm new to Vue.js and currently diving into using Vue.js router for my thesis project. However, I'm a bit confused about the terms "component" and "view". Could anyone help clarify this distinction? Additionally, if you have any tips or best practices for structuring a project with Vue.js router, I would greatly appreciate it!

**Comment by skirtles-code** [comment]
Views are normal Vue components, the term just indicates how those components are used.

It is used to refer to components used as route components. i.e. Components that are used in the `component` or `components` options of a route config. These are the components rendered using `<RouterView>`.

You may encounter projects with a `views` folder or with components that have names ending in `View`. These can be useful conventions, but the views themselves are still just components.

You might also encounter the term 'page' being used to mean the same thing as 'view'. Though context matters, as both 'view' and 'page' could be used to mean other things. e.g. View as in MVC and page as in webpage.

**Reply by chanezAMK** [reply]
Thank you for your response.
